packages/slice-machine/pages/slices.tsx

```diff
@@ -42,17 +44,22 @@ const CreateSliceButton = ({
 );
 
 const SlicesIndex: React.FunctionComponent = () => {
-  const libraries = useContext(LibrariesContext);
   const { openCreateSliceModal, closeCreateSliceModal, createSlice } =
     useSliceMachineActions();
 
-  const { isCreateSliceModalOpen, isCreatingSlice, localLibs, remoteLibs } =
-    useSelector((store: SliceMachineStoreType) => ({
-      isCreateSliceModalOpen: isModalOpen(store, ModalKeysEnum.CREATE_SLICE),
-      isCreatingSlice: isLoading(store, LoadingKeysEnum.CREATE_SLICE),
-      localLibs: getLibraries(store),
-      remoteLibs: getRemoteSlices(store),
-    }));
+  const {
+    isCreateSliceModalOpen,
+    isCreatingSlice,
+    localLibs,
+    remoteLibs,
+    libraries,
```

Comment:
- Duplicated data between `libraries` and `localLibs`
- Can you rename remoteLibs into remoteSlices.
  - I know it's not you, it's just wrong 😓

Reply:
libraries and localLibs are not exactly the same since one is a LibraryUI[] and the other is a LibraryState[]. Kept the `libraries` since that's the one that has more information 👍🏼